linera-views/src/collection_view.rs

```diff
@@ -119,14 +118,19 @@ where
             }
         }
         let hash = *self.hash.get_mut();
-        if self.stored_hash != hash {
+        // In tne admittedly rare scenarion that we do a clear
```

Comment:
fix spelling: `tne` `scenarion`

`scenario that` -> `scenario where` (I think?)

Reply:
nit: remove "admittedly rare" (in technical documentation, the shorter the better)

Reply:
Yes, for `scenario where`.